app/renderer/src/containers/result-list-container/result-list-container.js

```diff
@@ -47,17 +52,20 @@ const ResultListContainer = class extends Component {
       } else {
         resetResults();
       }
+      this.setState({ copiedToClipboard: false });
     });
     ipcRenderer.on(IPC_SELECT_PREVIOUS_ITEM, () => {
       if (self.props.selectedIndex > 0) {
         selectPreviousItem();
+        this.setState({ copiedToClipboard: false });
```

Comment:
Shouldn’t this be `self.setState`? :wink: 

Maybe we remove self while we’re at it?